src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptClientSideFormatOnStatementTerminatorTest.java

```diff
@@ -176,7 +176,29 @@ export class Foo {
         );
     }
 
-    public void testSimpleEnabled() {
+    public void testSimpleEnabled_statementTerminatorCharacters() {
```

Comment:
This looks like a bigger change than it is, but it now tests the statement terminator characters being defined at the language grammar-level and overridden at the format-on-statement terminator-level.